### ISSUE TITLE
Add possibility to give tip to devs even after buying a game

### ISSUE THREAD

**Issue by lebris** [leafo/itch.io#254]
When I'm buying a game I don't always know if I will like the game enough to give a tip. 
But after some hours playing, I sometime want to to it.

**Comment by ilexp** [comment]
Would appreciate this as well. Pay-what-you-want just makes most sense when the player already knows exactly what he is paying for; and small games don't always have the capacity to release a demo or extensive trailer (for spoiler- and other practical reasons).

**Comment by Deozaan** [comment]
Agreed. I thought this was how it worked. How else can people increase their payment to qualify for "pay at least X to unlock this download" kind of purchases?

**Comment by Deozaan** [comment]
Actually this just came up for me in a very real situation.

I sell a utility on Itch with a "Lite" and "Pro" version. People can pay the minimum amount for the Lite version, or pay a bit extra for the Pro version. I thought customers could just return to the product page and pay again to increase their payment amount to be above the threshold required to be eligible for the Pro version. But I guess the payment button is just replaced with download links?

I have a customer emailing me asking how they can upgrade to Pro after having purchased the Lite version. So I need to ask @leafo or @fasterthanlime what I'm supposed to do to allow this person to buy the Pro version as a discounted upgrade price.

**Comment by leafo** [comment]
@Deozaan We want to add an option to add more to a purchase, and that would allow them to upgrade tiers. Not sure when this will happen though. In your particular case, we can refund their Lite purchase and let them re-buy as Pro. Or you can just have them PayPal you directly and give them a fresh download key that unlocks it.

**Comment by Deozaan** [comment]
Thanks, @leafo.

> Or you can just have them PayPal you directly and give them a fresh download key that unlocks it.

But if I understand correctly, download keys don't unlock bonus downloads that unlock above the minimum purchase price. Is that correct?

**Comment by leafo** [comment]
if you create a bulk key group you can specify the unlock price: 

![bulk key dialog](http://leafo.net/shotsnb/2016-01-06_19-30-15.png)

Just pick a quantity of 1. I haven't ported over that unlock price logic to the old key system yet.

**Comment by Geonjaha** [comment]
I'd also appreciate this feature - having to buy a new product instead of increasing the amount you've donated is a bit of an annoyance.

**Comment by OhiraKyou** [comment]
This should have a much higher priority. Without upgrade options, the system seems excessively unfriendly to games that release additional content (e.g., expansions) over time.

I want to allow users to download a demo, lite, early, or base version at any or a reduced price. Then, when I release the full game or extra content for the base game, I want users to be able to upgrade to the full or expanded version tier from any previous version/tier.

This isn't a particularly unusual business model, and it should be supported at itch.io.

**Comment by Deozaan** [comment]
A new, higher-profile use-case for this feature is [Kenney](https://kenney.itch.io/)'s [Asset Forge](http://assetforge.io/).

He has created two store listings, [one for Asset Forge](https://kenney.itch.io/assetforge), and [another for Asset Forge Deluxe](https://kenney.itch.io/assetforge-deluxe), all in an effort [to allow people to upgrade from one to the other](https://twitter.com/Deozaan/status/881676208848519170).

This requires twice the maintenance (updating store listing info, downloads, emailing buyers, etc.), and some trickery with links to keep the product's community unified.
